src/ui/drawing/gdi/GDIBitmapSurface.hh

```diff
@@ -15,10 +15,12 @@ namespace gdi {
 class GDIBitmapSurface : public GDISurface
 {
 public:
+    GSL_SUPPRESS(type.6) // m_hBitmap not initizalied, adhered to note below
     explicit GDIBitmapSurface(const int nWidth, const int nHeight, ResourceRepository& pResourceRepository) noexcept :
         GDISurface(CreateBitmapHDC(nWidth, nHeight), RECT{0, 0, nWidth, nHeight}, pResourceRepository)
     {}
```

Comment:
I saw the comment near the data members so I suppressed the rules for these constructors.